src/renderer/components/pages/FilterPage.tsx

```
@@ -0,0 +1,373 @@
+    cancelledRef.current = false;
+    setPhase('running');
+    setProgress(null);
+    setResult(null);
+    setError(null);
+    try {
+      const res = await window.electronAPI.filterLibrary({
+        operationId, tracks, libraryPath: libraryPath ?? '',
+        filters: validRules,
+        options: { destination, mode, conflictResolution },
+      });
+      if (cancelledRef.current) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)